doc/developers/performance.rst

```diff
@@ -211,41 +211,18 @@ It is however still interesting to check what's happening inside the
 ``_nls_subproblem`` function which is the hotspot if we only consider
 Python code: it takes around 100% of the accumulated time of the module. In
 order to better understand the profile of this specific function, let
-us install ``line-prof`` and wire it to IPython::
+us install ``line-profiler`` and wire it to IPython::
```

Comment:
pip doesn't mind either way, but below we have `memory_profiler`, not `memory-profiler`. Can we choose consistently between `-` and `_`?